<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDialog</class>
 <widget class="QDialog" name="KeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>585</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>585</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Key</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="m_keyEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="m_spendKeyLabel">
     <property name="text">
      <string>Spend Secret Key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_spendKeyLineEdit">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="m_viewKeyLabel">
     <property name="text">
      <string>View Secret Key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="m_viewKeyLineEdit">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="WalletGui::WalletSmallGrayTextLabel" name="m_descriptionLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="WalletGui::WalletNormalBlueButton" name="m_fileButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="WalletGui::WalletCancelButton" name="m_cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WalletGui::WalletOkButton" name="m_okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::WalletOkButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Common/WalletOkButton.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletCancelButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Common/WalletCancelButton.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletSmallGrayTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletNormalBlueButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Common/WalletBlueButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_fileButton</sender>
   <signal>clicked()</signal>
   <receiver>KeyDialog</receiver>
   <slot>fileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>KeyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>KeyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_keyEdit</sender>
   <signal>textChanged()</signal>
   <receiver>KeyDialog</receiver>
   <slot>keyChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_spendKeyLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>KeyDialog</receiver>
   <slot>keysChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_viewKeyLineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>KeyDialog</receiver>
   <slot>keysChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileClicked()</slot>
  <slot>keyChanged()</slot>
  <slot>keysChanged()</slot>
 </slots>
</ui>
